graphql_schema.json

```diff
@@ -504,7 +504,7 @@
             {
               "name": "publicKey",
               "description":
-                "Public key you wish to start snark-working on; null to stop doing any snark work",
+                "Public key you wish to start snark-working on; null to stop doing any snark work. Warning: Ensure this address is a non-zkApp account or has the permission to receive funds set as None",
```

Comment:
Since this shows up as a tooltip, it's good to keep it terse. How about:
```
If the key is from a zkApp account, the account's receive permission must be None.
```